EKF/common.h

```diff
@@ -81,8 +81,8 @@ struct outputSample {
 };
 
 struct outputVert {
-	float	    vel_d;		///< D velocity calculated using alternative algorithm (m/sec)
-	float	    vel_d_integ;	///< Integral of vel_d (m)
+	float	    vert_vel;		///< Vertical velocity calculated using alternative algorithm (m/sec)
```

Comment:
it seems a bit redundant to me to specify that the velocity is the vertical one in a struct dedicated to the vertical axis

Reply:
Agree, had the same impression.